test/libsolidity/ASTJSON/address_payable.json

```diff
@@ -277,7 +277,7 @@
 											"name" : "this",
 											"nodeType" : "Identifier",
 											"overloadedDeclarations" : [],
-											"referencedDeclaration" : 65,
+											"referencedDeclaration" : 66,
```

Comment:
Where is this coming from?

Reply:
It is weird, because there is no such declaration, neither with id 66 nor with 65

Reply:
Ah right - this is about `this`. I added a new global variable `type`, which probably changed the order of assignments of IDs to global variables.

Reply:
Ah ok, that explains it.